app/views/components/daily_statements_v2.scala.html

```diff
@@ -47,7 +47,7 @@
                                <span class="hmrc-responsive-table__heading" aria-hidden="true">
                                    ${messages("cf.cash-account.detail.transaction-type")}
                                </span>
-                                ${if(statement.transactionType.mrnLink.isDefined) statement.transactionType.mrnLink.get else statement.transactionType.textString.getOrElse(emptyString)}
+                               ${statement.transactionType.flatMap(_.mrnLink).getOrElse(statement.transactionType.flatMap(_.textString).getOrElse(emptyString))}
```

Comment:
Could we please put this .getOrElse on another line? Something like below or how you like it

```suggestion
${statement.transactionType.flatMap(_.mrnLink)
.getOrElse(statement.transactionType.flatMap(_.textString).getOrElse(emptyString))}
```